doc/tools/nordic_segger.rst

```diff
@@ -78,6 +78,38 @@ and ``<x>`` is either 1 for nRF51-based boards or 2 for nRF52-based boards
 
 Where ``<x>`` is either 1 for nRF51-based boards or 2 for nRF52-based boards
 
+Debugging with GDB
+******************
+
+**Note**: The instructions in this section are common to all Operating Systems.
+
+Once the application is flashed into the board, you can opt to debug using GDB
+or try a visual debugger such as Segger Ozone, descibed in :ref:`nordic_segger_ozone`.
+
+* Start the J-Link GDB server on one console:
+
+.. code-block:: console
```

Comment:
This code block directive and the line below saying "Where..." should be indented by three spaces to be part of the step.